src/Form.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './Form.css'
+
+
+const Form = () => {
```

Comment:
This will need to be a controlled form and hold state for the search functionality.